### DISCUSSION TITLE
Blur effect per layers

### DISCUSSION THREAD

**Discussion by yuntan** [visgl/deck.gl#6299] [Q&A]
I want to apply [glow effect](https://blog.mapbox.com/glow-effect-the-firefly-technique-23eff7297075) to deck.gl's `TripsLayer` and `ScatterplotLayer`. Glow effect is a cartographic technique to brighten points and lines by stacking blurry layers and sharp layer. However, `PostProcessEffect` instance is only applicable to `Deck` (or `DeckGL`) instance, and I couldn't find any ways to apply blur effect per each layers. Did anyone know about ways to achive this?

related: #4155 #4440 #5653

**Comment by ismaelheinen** [comment]
I also trying to implement some similar effect, using additional layer with similar setup, but largest width and more transparency, but need some specific attribute to give blur to this second layer, that just with this i´m not getting expected result

**Comment by lianzhao** [comment]
Currently, I'm using "gradient border" to mimic glow/shadow effect. Something like [this](https://github.com/visgl/deck.gl/discussions/7372).
